src/index.js

```diff
@@ -5,7 +5,8 @@ function toVal(mix) {
 		str += mix;
 	} else if (typeof mix === 'object') {
 		if (Array.isArray(mix)) {
-			for (k=0; k < mix.length; k++) {
+			var len=mix.length, k=0;
+			for (; k < len; k++) {
```

Comment:
```suggestion
			var len=mix.length;
			for (k = 0; k < len; k++) {
```

Extracting `k` to the outer level is not required. The performance boost comes from caching `length`.

Reply:
Right! It's same as `for (var len=mix.length, k=0; k < len; k++) {`